en/asgardeo/docs/guides/users/migrate-users-to-asgardeo/migrate-passwords.md

```diff
@@ -0,0 +1,215 @@
+
+        ![Make the polling endpoint open]({{base_path}}/assets/img/guides/users/migrate-users/polling-make-open-endpoint.png)
+
+    3. Click **Apply** to save the changes.
+
+11. On the left navigation, go to **Manage** > **Lifecycle** and click **Publish**.
+
+### Step 3: Create a Choreo application to consume the APIs
+
+After the RESTful service is deployed, follow the steps below to create an application in Choreo and subscribe it to the REST API. This application will later be used to integrate the REST APIs with the Asgardeo application.
+
+1. Sign in to the [Choreo Developer Portal](https://devportal.choreo.dev/){target="_blank"} with the same email address you used to log in to the Choreo Console in step 2.
+
+2. On the top navigation, go to **Applications** and click **Create Application**.
+
+3. Provide a name and a description for the application and click **Create**.
+
+4. On the left navigation, go to **Credentials** and click **Generate Credentials**.
+
+5. Take note of the generated consumer key and consumer secret as you will need it later.
+
+6. On the left navigation, go to **Subscriptions** and click **Add APIs**.
+
+7. Click **Add** corresponding to the API that you created in step 2.
+
+!!! note
+
+    To learn more about applications in Choreo, refer to the [Choreo documentation](https://wso2.com/choreo/docs/consuming-services/manage-application/#create-an-application){target="_blank"}.
+
+
+### Step 4: Configure an attribute in Asgardeo to hold the migration status
+
+Asgardeo should have means to check whether a user's password is migrated or not. This can be achieved by creating a user attribute in Asgardeo to hold the migration status. To do so,
+
+1. On the Asgardeo Console, go to **User Attributes & Stores** > **Attributes** and click on **Attirubtes**.
+
+2. Click **New Attribute** and enter the following details: 
+
+    <table>
+        <tr>
+            <td><b>Attribute name</b></td>
+            <td>is_migrated</td>
+        </tr>
+        <tr>
+            <td><b>Attribute Display Name</b></td>
+            <td>Password migration status</td>
+        </tr>
+    </table>
+
+3. Click **Finish** to create the attribute.
+
+### Step 5: Configure the login flow of the Asgardeo application
+
+You are now ready to configure your application for on-demand silent password migration. To do so,
+
+1. On the Asgardeo Console, click **Application**.
+
+2. Select the application and go to its **Login Flow** tab.
+
+3. Configure the login flow using your preferred editor:
+
+    === "Classic Editor"
+
+        To configure the login flow using the classic editor:
+
+        1. Click **Start with default configuration** or add the **Username & Password** authenticator to the first step of the flow.
+        
+        2. Turn on Conditional Authentication by switching on the toggle.
+
+            ![Make the polling endpoint open]({{base_path}}/assets/img/guides/users/migrate-users/migrate-users-classic.png)
```

Comment:
```suggestion
            ![Configure with classic editor]({{base_path}}/assets/img/guides/users/migrate-users/migrate-users-classic.png)
```

Reply:
Fixed with https://github.com/wso2/docs-is/commit/d0b7130af4b2f0857f356c6dbdcbce192e8484c2